sklearn/decomposition/_fastica.py

```diff
@@ -334,9 +329,17 @@ class FastICA(_ClassNamePrefixFeaturesOutMixin, TransformerMixin, BaseEstimator)
     algorithm : {'parallel', 'deflation'}, default='parallel'
         Apply parallel or deflational algorithm for FastICA.
 
-    whiten : bool, default=True
-        If whiten is false, the data is already considered to be
-        whitened, and no whitening is performed.
+    whiten : str or bool, default="warn"
+        Specify the whitening strategy to use.
+        If 'arbitrary-variance' (default), a whitening with variance arbitrary is used.
+        If 'unit-variance', the whitening variance is adjusted to be unitary.
```

Comment:
Based on my understanding, the following would be both more explicit and correct because I am not sure how "whitening variance" is meant to be defined and I supposed that many readers can be confused as well.

```suggestion
        If 'unit-variance', the whitening matrix is rescaled to ensure that the
        recovered sources each have unit variance.
```